src/framework/preferences/css/stylus/utils/Fonts.styl

```diff
@@ -0,0 +1,81 @@
+// Contains utility functions for changing font family
+// !important is used for all styles to ensure that Preferences Framework's enactors are able to override the pages default styles.
+
+fonts = {
+    ".fl-font-arial": {
+        fontFamily: "Arial"
+    }
+    ".fl-font-verdana": {
+        fontFamily: "Verdana"
+    }
+    ".fl-font-times": {
+        fontFamily: "Georgia, Times, Times New Roman, serif"
+    }
+    ".fl-font-comic-sans": {
+        fontFamily: "Comic Sans MS, sans-serif"
+    }
+}
+
+build-fonts-prefsEditor(textFont) {
+
+	for fontName in textFont {
+		font = textFont[fontName].fontFamily;
+
+		.fl-prefsEditor {
+			{fontName},
+			{fontName} * {
+				:not([class*='icon']) {
+					font-family: unquote(font) !important;
+                }
+			}
+		}
+	}
+}
+
+build-fonts-separatedPanelPrefsEditorFrame(textFont) {
+
+    for fontName in textFont {
+        font = textFont[fontName].fontFamily;
+
+        .fl-prefsEditor {
+            {fontName},
+            {fontName} * {
+                :not([class*='icon']) {
+                    font-family: unquote(font) !important;
+                }
+            }
+        }
+    }
+}
+
+
+build-fonts-separatedPanelPrefsEditor(textFont) {
+
+    for fontName in textFont {
+        font = textFont[fontName].fontFamily;
+
+        .fl-prefsEditor {
+            {fontName},
+            {fontName} * {
+                :not([class*='icon']) {
+                    font-family: unquote(font) !important;
+                }
+            }
+        }
+    }
+}
+
+
+build-fonts-Enactors(textFont) {
```

Comment:
`build-fonts-prefsEditor`, `build-fonts-separatedPanelPrefsEditorFrame`, `build-fonts-separatedPanelPrefsEditor` and `build-fonts-Enactors` all appear to be the same. You should be able to reuse a single function in multiple places. Also similar to our JS coding style, you should probably leave a space between the function name and the arguments block. Like:

```
build-fonts-Enactors (textFont) {}
```